cratedb_mcp/__main__.py

```
@@ -12,8 +12,11 @@
 # Create FastMCP application object.
 mcp: FastMCP = FastMCP(__appname__)
 
-
 def query_cratedb(query: str) -> list[dict]:
+    """Sends a `query` to the set `$CRATEDB_CLUSTER_URL`"""
+    if HTTP_URL.endswith('/'):
```

Ruff (0.11.9)

Local variable `HTTP_URL` referenced before assignment

(F823)